pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>com.github.willowtreeapps</groupId>
     <artifactId>conductor-mobile</artifactId>
-    <version>0.10.1</version>
+    <version>0.10.2</version>
```

Comment:
I would like to follow semver a little closer. Since this is a new feature and not a bugfix the next version would be 0.11.0: https://semver.org/